src/sentry/incidents/endpoints/serializers.py

```diff
@@ -34,6 +34,7 @@
 from sentry.models.team import Team
 from sentry.models.user import User
 from sentry.snuba.models import QueryAggregations
```

Comment:
Can't kill this yet right?

Reply:
Not yet, we'll need to keep this around until we stop showing the old style ui, I think. We might be able to figure out a way to remove it sooner.